uaclient/cli/vulnerability/__init__.py

```diff
@@ -0,0 +1,21 @@
+from uaclient import messages
+from uaclient.cli.commands import ProCommand
+from uaclient.cli.vulnerability.list import list_subcommand
+from uaclient.cli.vulnerability.show import show_subcommand
+from uaclient.cli.vulnerability.update import update_subcommand
+
+
+def action_vulnerability(args, *, cfg, **kwargs):
+    # Avoiding a circular import
+    from uaclient.cli import get_parser
+
+    get_parser().print_help_for_command("vulnerability")
+
+
+vulnerability_command = ProCommand(
+    "vulnerability",
+    help=messages.CLI_ROOT_VULNERABILITY,
+    description=messages.CLI_VULNERABILITY_DESC,
```

Comment:
Descriptions are treated in `argparse` as a single paragraph of text (more or less). There is specific formatting though. 
Right now, `pro vulnerability --help` is shown as:

```
pro vulnerability --help
usage: pro vulnerability [-h] {show,update,list} ...

Show information about system vulnerabilities. show: display all available
information for a given CVE/USN. list: display all fixable CVEs in the system
update: update the vulnerability data in the system

options:
  -h, --help          show this help message and exit

Available Commands:
  {show,update,list}
    show              show information about a vulnerability
    update            update the vulnerability data in your machine
    list              list the vulnerabilities that affect the system
```

The thing is - if you want this description to be formatted as it is defined, you need to set `preserve_description`.

However, see that the list of options is present in `Available Commands:` in the lower text. So I would just keep the first phrase, for simplicity.

Reply:
On that - I think we should have the `help` integration test include this command and its subcommands - check how `config` is there for reference.